scripts/installers/install_opengrm.sh

```diff
@@ -0,0 +1,10 @@
+# Miniconda3
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+# Opengrm
+/root/miniconda3/bin/conda install -c conda-forge ngram -y
```

Comment:
Again, can we use the default python path?
Is conda and miniconda3 necessary for this tool?

Reply:
We could, but this is default conda path, usually it is added to PATH environment variable

Reply:
I tried to install without conda but I couldn't handle it

Reply:
Would you please add a note on the top that it just works with this conda?
It is going to install a new conda for the user, not to mention that it assumes some stuff like the architecture, access to root, etc.
@titu1994 would you please review these installation scripts?

Reply:
Just installed Miniconda on the previous lines of this file

Reply:
No please don't install any kind of conda in our containers.

Reply:
And if you have to, do not include this script for installation inside of the Nemo dockerfile. No conda is allowed there. Users must manually run and build this file.

Reply:
sure, it is not a part of Nemo dockerfile, it is an option on the top of it